app/main/views/withdraw_brief.py

```diff
@@ -0,0 +1,32 @@
+from flask import abort, flash, redirect, url_for
+from flask_login import current_user
+
+from app import data_api_client
+from .. import main
+from ..helpers.buyers_helpers import (
+    get_framework_and_lot,
+    is_brief_correct,
+)
+
+
+BRIEF_WITHDRAWN_MESSAGE = "You’ve withdrawn your requirements for ‘{brief[title]}’"
```

Comment:
This line is still in `views.buyers`, I think

Reply:
Good spot!